Package.swift

```diff
@@ -11,6 +11,7 @@ let package = Package(
         .library(name: "Orbit", targets: ["Orbit"]),
         .library(name: "OrbitDynamic", type: .dynamic, targets: ["Orbit"]),
         .library(name: "OrbitStatic", type: .static, targets: ["Orbit"]),
+        .library(name: "OrbitStorybook", targets: ["OrbitStorybook"]),
```

Comment:
In a test project that uses both Orbit and OrbitStorybook now, there are probably some naming collisions. I will try to dig deeper, but maybe you will have an idea why it might happen:

<img width="1442" alt="image" src="https://user-images.githubusercontent.com/35844477/198819051-5152ff4b-5717-4d57-aeb4-79493461cde0.png">

Reply:
I tried that with test app and this didn't happen. 

<img width="817" alt="Screenshot 2022-10-29 at 10 45 04" src="https://user-images.githubusercontent.com/12349477/198822492-1110f62b-0098-4c55-9c31-f1126b241d2c.png">

<img width="826" alt="Screenshot 2022-10-29 at 10 44 45" src="https://user-images.githubusercontent.com/12349477/198822454-d483c218-545f-458b-9f1f-6e1d7d77ef19.png">

Reply:
Probably something wrong in my project